### ISSUE TITLE
Installation of Inkscape fails on macOS 10.14 

### ISSUE THREAD

**Issue by adamsloma** [caskformula/homebrew-caskformula#83]
Error: Failed to import: /usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula/Formula/inkscape.rb
Calling needs :cxx11 is disabled! There is no replacement.

**Comment by gokit1** [comment]
Ditto (Mac OS 10.14.3)
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula/Formula/inkscape.rb
Calling needs :cxx11 is disabled! There is no replacement.
Please report this to the caskformula/caskformula tap:
  /usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula/Formula/inkscape.rb:49

Fixed by:
  brew tap homebrew/cask
  brew cask install inkscape

**Comment by danyelf** [comment]
This is a duplicate of https://github.com/caskformula/homebrew-caskformula/issues/81

**Comment by qwik** [comment]
same issue for me

**Comment by geoeo** [comment]
same issue for me. cant install inkscape. brew update / upgrade works fine.

**Comment by dg01d** [comment]
Installation succeeds for me in 10.14, but then fails spectacularly on startup.

I receive the attached errors on command-line; I also attach copy of the incredibly helpful and informative XQuartz error dialog
<img width="660" alt="xquartz error" src="https://user-images.githubusercontent.com/1061809/55876067-a13b1c00-5b8e-11e9-831c-7e7726e7b25c.png">

[inkscapeError.txt](https://github.com/caskformula/homebrew-caskformula/files/3063565/inkscapeError.txt)

**Comment by inclusive-dev1** [comment]
> Installation succeeds for me in 10.14, but then fails spectacularly on startup.
> 
> I receive the attached errors on command-line; I also attach copy of the incredibly helpful and informative XQuartz error dialog
> [inkscapeError.txt](https://github.com/caskformula/homebrew-caskformula/files/3063565/inkscapeError.txt)

Same for me. I'm not sure this is something homebrew can fix. I downloaded the `.dmg` for [0.92.2](https://inkscape.org/release/0.92.2/mac-os-x/). Same problem.

Seems Inkscape doesn't work with Mojave.

**Comment by david-bonin** [comment]
> Ditto (Mac OS 10.14.3)
> Error: Invalid formula: /usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula/Formula/inkscape.rb
> Calling needs :cxx11 is disabled! There is no replacement.
> Please report this to the caskformula/caskformula tap:
> /usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula/Formula/inkscape.rb:49
> 
> Fixed by:
> brew tap homebrew/cask
> brew cask install inkscape

Solid solution!

**Comment by thisbailiwick** [comment]
> Solid solution!
@david-bonin 

Were you able to open it? It showed as installed on cli but when I try to open it...nada.

**Comment by david-bonin** [comment]
Yeah, it worked for me. I might have tweaked something though I don't remember.

**Comment by dehesselle** [comment]
Can you check if this helps: https://github.com/Homebrew/homebrew-cask/issues/62404#issuecomment-497327144

**Comment by thisbailiwick** [comment]
> Can you check if this helps: [Homebrew/homebrew-cask#62404 (comment)](https://github.com/Homebrew/homebrew-cask/issues/62404#issuecomment-497327144)

@dehesselle That seemed to do the trick! I don't think I have touched anything regarding this since my last post so your reference to the other comment was my fix, or I guess more, part of it. 

Thanks so much!

**Comment by jepsar** [comment]
> ...
> Fixed by:
> brew tap homebrew/cask
> brew cask install inkscape

It worked for me, but required a reboot

**Comment by jediunixmaster** [comment]
Hello.
Macbook Pro 10.14.5
 I followed this thread as I was having issues as well. I brew installed xquartz and then restarted my machine as suggested by @jepsar and I then tried to install inkscape with the following commandline copy: 

> brew tap homebrew/cask
> brew cask install inkscape  
==> Satisfying dependencies
==> Downloading https://media.inkscape.org/dl/resources/file/Inkscape-0.92.2-1-x11-10.7-x86_64.dmg
######################################################################## 100.0%
==> Verifying SHA-256 checksum for Cask 'inkscape'.
==> Installing Cask inkscape
==> Moving App 'Inkscape.app' to '/Applications/Inkscape.app'.
==> Linking Binary 'inkscape' to '/usr/local/bin/inkscape'.
🍺  inkscape was successfully installed!

> brew list 
asdf			coreutils		libyaml			nvm			postgresql		zsh
autoconf		icu4c			mongodb-community	openssl			readline
automake		libtool			ncurses			pcre			unixodbc

> brew cask list 
inkscape                                                     iterm2                                                       xquartz

> brew install caskformula/caskformula/inkscape
==> Tapping caskformula/caskformula
Cloning into '/usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula'...
remote: Enumerating objects: 5, done.
remote: Counting objects: 100% (5/5), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 5 (delta 0), reused 3 (delta 0), pack-reused 0
Unpacking objects: 100% (5/5), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula/Formula/inkscape.rb
inkscape: "cxx11" is not a recognized standard
Error: Cannot tap caskformula/caskformula: invalid syntax in tap!


Maybe I've missed something?
Thank you for your help with this.

**Comment by henngelm** [comment]
> brew cask install inkscape

Cask 'inkscape' requires XQuartz/X11, which can be installed using Homebrew Cask by running:
  brew cask install xquartz

**Comment by jediunixmaster** [comment]
Hi henngelm
ya, I mentioned I did that in te first line. Its just not in the console logging because i restarted my machine and that got wiped. But, you can see it where I did a brew list cask.

**Comment by okao** [comment]
> Ditto (Mac OS 10.14.3)
> Error: Invalid formula: /usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula/Formula/inkscape.rb
> Calling needs :cxx11 is disabled! There is no replacement.
> Please report this to the caskformula/caskformula tap:
> /usr/local/Homebrew/Library/Taps/caskformula/homebrew-caskformula/Formula/inkscape.rb:49
> 
> Fixed by:
> brew tap homebrew/cask
> brew cask install inkscape

thanks. this is a working solution

**Comment by bhayes-zd** [comment]
> Fixed by:
brew tap homebrew/cask
brew cask install inkscape

Supported brew command is now:
```shell
brew tap homebrew/cask
brew install --cask inkscape
```
